NitroxClient/GameLogic/PlayerGameObjectManager.cs

```diff
@@ -17,6 +20,30 @@ public void UpdatePlayerPosition(string playerId, Vector3 position, Quaternion r
             MovementHelper.MoveGameObject(player, position, rotation, PLAYER_TRANSFORM_SMOOTH_PERIOD);
         }
 
+        public void UpdateAnimation(string playerId, AnimChangeType type, AnimChangeState state)
+        {
+            GameObject player = GetPlayerGameObject(playerId);
+            GameObject playerView = player.transform.Find("player_view").gameObject;
+            AnimationController controller = playerView.GetComponent<AnimationController>();
+
+            bool value;
+            if (state == AnimChangeState.Off)
+            {
+                value = false;
+            }
+            else
+            {
+                value = true;
+            }
+
+            switch (type)
+            {
+                case AnimChangeType.Underwater:
+                    controller.SetBool("is_underwater", value);
+                    break;
+            }
+        }
+
         public void HidePlayerGameObject(string playerId)
```

Comment:
I recommend refactoring this - perhaps breaking this up into helper functions and avoiding magic values.